### DISCUSSION TITLE
[매모판 Pro] 솔루션 관리하기 - 'API그룹 추가 사유' 수정 불가

### DISCUSSION THREAD

**Discussion by prami79** [commerce-api-naver/commerce-solutions#67] [묻고 답하기] [answered]
안녕하세요, 매모판 팀입니다. 

솔루션 요금제 관련 변경이 필요한 상황인데,
API그룹 추가 사유 수정 불가현상이 있습니다. 

지난 8월에 동일 이슈가 있어
그때는 제휴 담당자 분께 요청드려 솔루션 정보 수정을 진행했는데요.

당시에 이 이슈가 픽스될 것으로 기대했었는데
아직 동일 현상이 있습니다. 
![image](https://github.com/user-attachments/assets/218c6cd7-9eed-4474-a386-40a32a169f5c)

확인을 부탁드립니다.
감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

해당 내용의 확인을 위해 한가지 여쭙습니다.
지난 8월과 동일하게 아래와 같은 현상이 맞을까요?
- 'API 그룹 추가 사유' 입력란(`textarea`)이 비활성화 상태 - 입력 불가
- 솔루션 정보 수정 시도 시 유효성 검증으로 대상 입력란에 대해 필수 입력을 요구

#### [11:07 추가]
저희쪽에서는 재현이 어려운 것으로 보여 혹시 스크린샷이나 좀 더 자세한 현상을 알려주실 수 있을까요?

**Comment by prami79** [comment]
안녕하세요 민석님, 

네. 지난번과 동일한 현상이구요.

필수입력인 항목인데
요렇게 비활성화 되어 입력이 불가한 상태입니다.

![image](https://github.com/user-attachments/assets/e1f655e1-2bc1-44f6-ab16-b7e2da1465c6)

**Reply by commerce-api-naver** [reply]
@prami79 님, 커머스API 이민석입니다.

아시다시피 오픈된 솔루션은 'API 그룹 추가 사유'를 수정할 수 없는게 원칙인데요,
혹시 이 상태에서 솔루션 정보 수정 시도(페이지 최하단 '저장하기' 버튼 선택) 시 어떤 메시지가 표시되는지 알 수 있을까요?

**Reply by prami79** [reply, marked as answer]
앗, 민석님 죄송합니다~

테스트용 매모판 정보 수정을 통해 테스트 하다가
다른 부분 오류가 있었습니다.

API 그룹 추가 사유 아닌 것으로 확인되었어요.
번거롭게 해드려서 죄송합니다~!

도움 주셔서 감사합니다.
